py7zr/py7zr.py

```diff
@@ -37,7 +37,7 @@
 import sys
 from multiprocessing import Process
 from threading import Thread
-from typing import IO, Any, BinaryIO, Dict, List, Optional, Tuple, Type, Union
+from typing import IO, Any, BinaryIO, Dict, Collection, List, Optional, Tuple, Type, Union
```

Comment:
```suggestion
from typing import IO, Any, BinaryIO, Collection, Dict, List, Optional, Tuple, Type, Union
```
sort order is wrong.

Reply:
You can check it by `tox check`